source/type/stop_point.cpp

```diff
@@ -49,7 +50,7 @@ void StopPoint::serialize(Archive& ar, const unsigned int /*unused*/) {
     //
     // stop_point_connection_list is managed by StopPointConnection
     ar& uri& label& name& stop_area& coord& fare_zone& is_zonal& address_id& idx& platform_code& admin_list&
-        _properties& impacts& dataset_list& address;
+        access_points& _properties& impacts& dataset_list& address;
```

Comment:
The comment above says that serializing lists may create problems.
Is it complicated to do something like what is done for stop_point_connection_list ? 
It seems to be filled here : https://github.com/CanalTP/navitia/blob/1049b0a053b4d54b3ce2b9bd64ced9390bc42398/source/type/connection.cpp#L49-L51

Reply:
:+1: